src/common/SPELLS/encounter.js

```diff
@@ -1374,6 +1374,11 @@ export default {
     name: "Titan Spark",
     icon: "item_holyspark",
   },
+  ZEKVOZ_ENCOUNTER_EVENT: {
```

Comment:
Please add this as a constant to the top of the `Zekvoz.js` file. One of the long term goals is to drop the SPELLS files and only put the spells close to relevant places that also use it so that they can be code split properly. Looking at your usage, I reckon you can put it in the Zekvoz file just file since that should always be loaded when this spell object is relevant.

Reply:
Not related to the review, but while it makes sense somewhere, wouldn't that lead to a LOT of code duplication when spells are used in more places? especially for things like SpellLink etc.?

Reply:
It can be exported if it is used elsewhere. As for SpellLink I have no solution yet. This is a big reason why we're not there yet, but these spells don't appear to be spells we'll reuse.

Reply:
This ability has actually been used by a few bosses to trigger a phase change. I don't think it's going to be used in many places so I can certainly move it.

Reply:
The way chunking will work is each relevant part will be separate, so for boss phasing it will probably end up being so each boss is its own chunk. If it's a boss specific thing, it helps to be put with the boss specific stuff. If it's shared, it might belong in its own file or something that's shared. If it's going to be used with SpellLink then spells might make sense and we'll have to figure out a solution later.

Boss specific abilities seems like something that can grow big and will be mostly exclusive to that boss, so splitting it from the start would be very nice.